dist/main.js.LICENSE.txt

```diff
@@ -0,0 +1,8 @@
+/**
+ * @license
+ * Lodash <https://lodash.com/>
+ * Copyright OpenJS Foundation and other contributors <https://openjsf.org/>
+ * Released under MIT license <https://lodash.com/license>
+ * Based on Underscore.js 1.8.3 <http://underscorejs.org/LICENSE>
+ * Copyright Jeremy Ashkenas, DocumentCloud and Investigative Reporters & Editors
+ */
```

Comment:
### Your MIT.md :memo:
![MIT md](https://user-images.githubusercontent.com/93039359/184197701-cae0e765-71b3-4d2d-82db-ffd0c062e974.png)

- [ ] Your MIT.md file is empty. Means you didn't add your **name**, **current year**, and **app-type**. Please add these things :+1: This is the license file of your project so please take it seriously. :thinking: